common/app-starters-security-common/README.adoc

```diff
@@ -1,10 +1,19 @@
 === `App Starters Security` Common Module
 
-This artifact contains a Spring Boot auto-configuration to disable security in the application that set `spring.cloud.stream.security.enabled = false` (default).
-If the `spring.cloud.stream.security.enabled` property is explicitly set to `true` then this configuration falls back to the default Spring Security configuration.
+Spring Boot auto-configuration to manage the web security of the application starters.
+
+The `spring.cloud.stream-app-starters.security.enabled` and `spring.cloud.stream-app-starters.security.csrf-enabled` can customize adapter's behavior. Setting `spring.cloud.stream.security.enabled = false` would completely surpass the security for the application. For secured application setting `spring.cloud.app-starter-stream.security.csrf-enabled = false` would disable only the CSRF access.
```

Comment:
I'm not keen with `starters` at all.
Also I don't think that is is a good practice to have deshes in the prefix name.
In addition the `common` doesn't make sense as well. Since it is going to be a property for particular app, not so common throughout the stream in SCDF.

Plus pay attention, please, how you are missing one more to rename in this line.

Reply:
we are to decide on the prefix name later today